cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,29 +1,28 @@
 NAME
-    Ballerina build - Builds Ballerina module(s) and generates executable
+    Ballerina build - Build a Ballerina module(s)/file and generate executable
     outputs.
 
 SYNOPSIS
     ballerina build [-o <output>] [--sourceroot] [--off-line] [--skip-tests]
                     [--skip-lock] {<ballerina-file | module-name> | -a | --all}
 
 DESCRIPTION
-    Builds Ballerina module(s)/file and produces an executable jar file(s).
 
-    Building a Ballerina project or a specific module in a project the
-    executable .jar files will be created in <project-root>/target/bin directory.
+    Building a Ballerina project or a specific module in a project creates the
+    executable .jar files in the <project-root>/target/bin directory.
 
-    Building a single Ballerina file will create an executable .jar file in the
+    Building a standalone Ballerina file will create an executable .jar file in the
```

Comment:
@hevayo This is no longer the case right? Only `ballerina run` works with standalone files right?

Reply:
Now, the `build` command works for standalone files.